src/app/(BottomTabNavigation)/Profile/DeleteAccount/index.tsx

```diff
@@ -0,0 +1,46 @@
+//import { router } from 'expo-router';
+import React, { useEffect, useState } from 'react';
+import { Text, TouchableOpacity, View } from 'react-native';
+
+import {
+  deleteCurrentUser,
+  getCurrentUserInfo,
+  signOutUser,
+} from '../../../../supabase/queries/auth';
+import styles from '../styles';
+
+function DeleteAccountScreen() {
+  const [currSession, setCurrSession] = useState<any>(null);
```

Comment:
Let's try to not use the `any` type! We can make this a `User`! Just make sure to update the `User` type in the `types` file!

Reply:
Let me know if you disagree with this but since we know we will set the user in the `useEffect`, maybe we can have a default `userInstance` in `types.tsx` where all the fields are empty strings just so using `currSession` does not complain at us. Would look like this here: `const [currSession, setCurrSession] = useState<User>(userInstance);` and we could define `userInstance` in `types.tsx`